src/validations/rules/sap.sch

```diff
@@ -526,6 +528,13 @@
         id="metadata">
         <sch:rule
             context="oscal:metadata">
+            <sch:let
+                name="partyPersonIDs"
```

Comment:
This might better be named `SAP-partyPersonIDs`.

Reply:
Done